### ISSUE TITLE
Accessibility: active tab is differenciated only by color

### ISSUE THREAD

**Issue by Dehelssey** [BDX-town/elk#39]
The active tab in the bottom nav bar is only differenciated from the others by its color. It goes against accessibility good practices and active tab should be highlighted with an additional marker. Either the icon goes from linear to filled. Or a small indicator could be displayed underneath (a dot or a bar for instance)

I know you know this and maybe I should ask this directly on Elk's own repo. You know what, I am, right now !

But just in case, there it is anyway

**Comment by Cl0v1s** [comment]
Should be done on upstream!

**Comment by Dehelssey** [comment]
Already opened an issue there
